frontend/packages/noobaa-storage-plugin/src/components/details-card/details-card.tsx

```diff
@@ -63,6 +67,8 @@ export const ObjectServiceDetailsCard: React.FC<DashboardItemProps> = ({
   const queryResult = prometheusResults.getIn([NOOBAA_SYSTEM_NAME_QUERY, 'result']);
 
   const systemName = _.get(queryResult, 'data.result[0].metric.system_name');
+  const systemAddress = _.get(queryResult, 'data.result[0].metric.system_address');
+  const systemLink = `${systemAddress}fe/systems/${systemName}`;
```

Comment:
we need to handle the case when the `systemName` and `systemAddress` will be not defined and hence systemLink will be broken.
can refer https://github.com/openshift/console/blob/master/frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card.tsx#L70

Reply:
The `systemError` variable below should handle such case, unless I'm missing something.

The `systemLink` variable could be defined as

```ts
const systemLink = !systemError ? `${systemAddress}fe/systems/${systemName}` : undefined;
```

to make it clear that `systemLink` as a whole is undefined unless both `systemName` and `systemAddress` are valid.

Anyway, I'd suggest to simplify it further, just use

```ts
const systemLink = (systemName && systemAddress) ? `${systemAddress}fe/systems/${systemName}` : undefined;
```

and in JSX code below

```
error={!systemLink}
```